bedrock/mozorg/templates/mozorg/mpl/historical.html

```diff
@@ -36,10 +36,10 @@
             <dd>Since legal documents are difficult for non-lawyers to read, Mozilla has in the past provided a version of the license which contains informal explanations of various sections of the license, in the form of hyperlinks.  Note that <em>these explanations are not the license.</em> The explanations are not a legal document, or legal advice. If you need to know <em>exactly</em> what the license requires, you need to read and understand the license itself; if you need legal advice, you need to talk to a lawyer. The annotations are merely a good faith effort to explain the license in simpler language.</dd>
 
             <dt><a href="NPL/1.0/">Netscape Public License, version 1.0</a></dt>
```

Comment:
Not sure if these would need to be updated to archived url or if the redirect would work? Maybe no change is needed but I was a little confused about this when I updated these docs.